rabbitmq.go

```diff
@@ -97,7 +99,8 @@ func NewRabbitMqClient(connStr string, opt ...RabbitOption) (*RabbitMqClient, er
 		return nil, errPool
 	}
 	mc.pubChannelPool = pool
-
+	mc.done = make(chan bool, 1)
+	mc.signaldone = make(chan bool, len(mc.subscribers)*mc.threadNum)
```

Comment:
@bintangsibarani I think we don't need to create buffered channel fro `done` and `signaldone`. We are ok for blocking write in `done`. And for `signaldone` we don't need to write to it. We can just close the channel when we want to close the consumers